EventsExpress.Db/Migrations/20211101204509_AddUserMoreInfo.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace EventsExpress.Db.Migrations
+{
+    public partial class AddUserMoreInfo : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "UserMoreInfo",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    ParentStatus = table.Column<int>(nullable: false),
+                    RelationShipStatus = table.Column<int>(nullable: false),
+                    TheTypeOfLeisure = table.Column<int>(nullable: false),
```

Comment:
Are all these fields required to be completed while UserMoreInfo filling? Probably it makes sense to have it nullable